tests/GeneratorTest.php

```diff
@@ -147,4 +148,17 @@ public function canSetValidatorUrl()
         $this->assertTrue(file_exists($this->jsonDocsFile()));
         $this->assertTrue(file_exists($this->yamlDocsFile()));
     }
+
+    /** @test */
+    public function canAppropriateYamlType()
+    {
+        $this->setAnnotationsPath();
+
+        Generator::generateDocs();
+
+        $objects = Yaml::parse(file_get_contents($this->yamlDocsFile()), Yaml::PARSE_OBJECT_FOR_MAP);
```

Comment:
Could we use `(new Parser())->parse(...)` just to remove static call and make CodeClimate happier :)